roles/openshift_health_checker/library/rpm_version.py

```diff
@@ -106,7 +113,9 @@ def _check_pkg_versions(found_pkgs_dict, expected_pkgs_dict):
                 "The following packages were found to be installed with an incorrect version: {}".format('\n'.join([
                     "    \n{}\n    Required version: {}\n    Found versions: {}".format(
                         pkg_name,
-                        pkg["required_version"],
+                        ' or '.join(pkg["required_versions"]) if len(pkg["required_versions"]) < 3
```

Comment:
Suggestion: keep the message consistent by using either "or" or "," to concatenate lists of versions, otherwise it will look weird.

We may also s/version/versions in the message.